src/Libraries/Microsoft.Extensions.Hosting.Testing/Microsoft.Extensions.Hosting.Testing.csproj

```diff
@@ -26,7 +26,6 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Bcl.TimeProvider" />
-    <xPackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" />
```

Comment:
Good catch.

Reply:
whops, that's a leftover when I was trimming the dependency set across the repo.